sdk/eventhub/azure-eventhub/azure/eventhub/_producer.py

```diff
@@ -189,9 +190,10 @@ def _wrap_eventdata(
     ):
         # type: (...) -> Union[EventData, EventDataBatch]
         if isinstance(event_data, EventData):
```

Comment:
do we need to check if data is AmqpAnnotatedMessage here?
```suggestion
        if isinstance(event_data, EventData):
```

Reply:
no, I don't think it's needed since this method is only called in `send` in the same file, which is called by `send_batch`, which transforms `AmqpAnnotatedMessage` to `EventData` by the call to `send`. Since customers are only directed to use `send_batch` to send, they shouldn't call `send`.